### DISCUSSION TITLE
It is possible to search for issues in repositories with a specific topic?

### DISCUSSION THREAD

**Discussion by pawelmysior** [community/community#22907] [New to GitHub]
<p>Hello,</p>
<p>I’m trying to search for issues with the label <em>help wanted</em> within repositories that have a specific topic. I’ve tried the following search query: </p>
<pre><code>topic:laravel label:"help wanted"
</code></pre>
<p>but it doesn’t seem to work. It displays all issues accross GitHub that have the <em>help wanted</em> label.</p>
<p>Thanks in advance for any help.</p>
<p>Regards!</p>

**Comment by mpboom** [comment, marked as answer]
<p>Try</p>
<pre><code>is:open is:issue topic:"laravel" label:"help wanted"
</code></pre>
